behave_test_automation/pah_behave/features/environment.py

```diff
@@ -1,12 +1,27 @@
 from selenium import webdriver
 
+from features.page_objects.logout_page import LogoutPage
+from features.page_objects.login_page import LoginPage
+from features.page_objects.new_drive_page import AddNewDrivePage
```

Comment:
```suggestion
from features.page_objects.new_drive_page import NewDrivePage
```

You're breaking your own name convention here. Every class is named after it's file (`LogoutPage` -> `logout_page`, etc) but this here you've named the class `AddNewDrivePage` where file is named `new_drive_page`.

Reply:
Changes applied, file names adjusted